src/reachy_mini/daemon/app/services/bluetooth/install_service_bluetooth.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Get the directory where this script is located
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+SERVICE_NAME="reachy-mini-bluetooth"
+SERVICE_FILE="/etc/systemd/system/${SERVICE_NAME}.service"
+LAUNCHER_PATH="$SCRIPT_DIR/bluetooth_service.py"
+COMMANDS_DIR="$SCRIPT_DIR/commands"
+SERVICE_PATH=/bluetooth/bluetooth_service.py
+
+sudo cp "$LAUNCHER_PATH" "$SERVICE_PATH"
+sudo cp -r "$COMMANDS_DIR" /bluetooth/commands
+# Ensure Python script is executable
+sudo chmod +x "$SERVICE_PATH"
+
+# Create the systemd service file
+cat <<EOF | sudo tee "$SERVICE_FILE" > /dev/null
+[Unit]
+Description=Reachy Mini Bluetooth GATT Service
+After=network.target bluetooth.target
+Requires=bluetooth.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/python3 $SERVICE_PATH
```

Comment:
I don't have the bluetooth 100% of the time. I guess we should also set 
ExecStart=/bin/bash -c 'sudo /usr/sbin/rfkill unblock bluetooth && sleep 2 && /usr/bin/python3 /bluetooth/bluetooth_service.py'